### ISSUE TITLE
url not found

### ISSUE THREAD

**Issue by rofl0r** [binarymaster/3WiFi#77]
hey @binarymaster is there a new url for 3wifi ? the old one seems to be expired.
also if you want to have a chat, feel free to join sox and me in reaver chan ;)

**Comment by binarymaster** [comment]
@rofl0r yes, the server is down currently, and domain name doesn't belong to stascorp anymore.

Can you remind me the channel address?

**Comment by rofl0r** [comment]
`#reaver` on irc.libera.chat
